stories/core/Fieldset.stories.tsx

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React, { useState } from 'react';
+import {
+  Fieldset,
+  FieldsetProps,
+  InputGroup,
+  LabeledSelect,
+  Radio,
+  ToggleSwitch,
+} from '../../src/core';
+
+export default {
+  title: 'Input/Fieldset',
+  component: Fieldset,
+  argTypes: {
+    children: { control: { disable: true } },
+  },
+} as Meta<FieldsetProps>;
+
+export const Basic: Story<FieldsetProps> = (args) => {
+  const [value, setValue] = useState<number | undefined>(undefined);
+
+  return (
+    <Fieldset legend='General Settings' {...args}>
+      <LabeledSelect
+        label='Resolution'
+        options={[
+          { value: 1, label: '1200 x 1000' },
+          { value: 2, label: '1600 x 1200' },
+          { value: 3, label: '2560 x 1600' },
+        ]}
+        displayStyle='inline'
+        value={value}
+        onChange={(value) => setValue(value)}
+        placeholder='Select'
+      />
+
+      <InputGroup label='Color Theme' displayStyle='inline'>
+        <Radio name='choice' value='option1' label={'Light'} />
+        <Radio name='choice' value='option2' label={'Dark'} />
+        <Radio name='choice' value='option3' label={'Match device'} />
+      </InputGroup>
+
+      <InputGroup>
+        <ToggleSwitch label='Share usage statistics' />
+        <ToggleSwitch label='Share crash logs' />
+        <ToggleSwitch disabled label='Advanced settings' />
+      </InputGroup>
+    </Fieldset>
+  );
+};
+Basic.args = { legend: 'General Settings', disabled: false };
```

Comment:
Args that have the same value in each story, can be defined in global `args` like in `LabeledInput.story.tsx`:
![image](https://user-images.githubusercontent.com/36186912/128973542-d1dbc832-4a1c-47a2-9124-4ba77daa1e3a.png)

Reply:
`disabled` prop is false by default no need to define it here.

Reply:
Storybook does not infer default values anymore, so it's good to define `disabled` here